CODEOWNERS

```diff
@@ -31,3 +31,6 @@
 /solidity-v1/contracts/utils/PercentUtils.sol @Shadowfiend @pdyraga
 /solidity-v1/contracts/utils/UIntArrayUtils.sol @Shadowfiend @pdyraga
 /solidity-v1/package.json @Shadowfiend @pdyraga
+
+/solidity/random-beacon/contracts/RandomBeacon.sol @pdyraga @nkuba @lukasz-zimnoch
+/solidity/ecdsa/contracts/WalletRegistry.sol @pdyraga @nkuba @lukasz-zimnoch
```

Comment:
Let's extract `CODEOWNERS` change to a separate PR and merge it to `main`. 

The `CODEOWNERS` should cover two contracts listed here as well as all files in:
- `solidity/ecdsa/contracts/libraries` 
- `solidity/ecdsa/contracts/api`
- `solidity/random-beacon/contracts/libraries/`
- `solidity/random-beacon/contracts/api`

Let's add there all developers who worked on the Solidity code of the mentioned smart contracts: `@pdyraga @nkuba @lukasz-zimnoch @dimpar @tomaszslabon`.

This will not only protect dApp-dev-only changes from being merged but will also protect from any accidental changes to an already audited code.

Reply:
https://github.com/keep-network/keep-core/pull/3125